src/storage/context/StorageExpressionContext.cpp

```diff
@@ -92,7 +92,7 @@ Value StorageExpressionContext::getSrcProp(const std::string& tagName,
                                            const std::string& prop) const {
     if (!isEdge_ && reader_ != nullptr) {
         if (tagName != name_) {
-            return Value::kNullValue;
+            return Value::kEmpty;
```

Comment:
Line 114 perhaps need to change it as well.

Reply:
done